src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -6,15 +6,22 @@
  * Format full help instructions for every command for display.
  */
 public class HelpCommand extends Command {
-    public static final String ADD_COMMAND = "add c/CS1231 cr/4 t/Core : Adds module 'CS1231'"
-            + "with 4 credits and a 'Core' tag to it.";
+    public static final String ADD_COMMAND = "add CS1231 t/Core : Adds module 'CS1231' with 'Core' tag to it.";
     public static final String HELP_COMMAND = "help : Displays the help guide.";
-    public static final String LIST_COMMAND = "list : Displays list of current modules stored";
-    public static final String EDIT_COMMAND = "edit 1 c/CS1231 cr/4 :"
-            + "Replaces the current module at index 1 with module 'CS1231' with 4 credits.";
-    public static final String FIND_COMMAND = "find CS1231 : Searches for the 'CS1231' module.";
-    public static final String DELETE_COMMAND = "delete 1 : Deletes the module at index 1.";
+    public static final String LIST_COMMAND = "list : Displays list of current modules stored.";
+    public static final String EDIT_COMMAND = "edit CS1231 c/CS1101S :"
+            + "Replaces the module 'CS1231' with module 'CS1101S'.";
+    public static final String FIND_COMMAND = "find CS1 : Displays all modules that contain the prefix 'CS1'. \n"
```

Comment:
Actually we don't do prefix search 😁  searching "103" will return "CS2103T" also
```suggestion
    public static final String FIND_COMMAND = "find CS1 : Displays all modules that contain the keyword 'CS1'. \n"
```

Reply:
okay I think i seriously mixed up the definion of prefix HAHAHA